### ISSUE TITLE
BHĀGAVATAPURĀṆA literary source markup

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/PWK#109]
Standardize the markup for PW dictionary ([refer PWG work](https://github.com/sanskrit-lexicon/PWG/issues/81)).

1003 matches in 975 lines for "BHĀG. P." in buffer: pw.txt.

**Comment by funderburkjim** [comment]
@Andhrabharati   Here are 5 BHĀG. P. references in PW(K) that don't match the index for Burnouf version.
```
21824	ekapati	84708	<ls>BHĀG. P. 4,26,27</ls>	     27 not found
49440	daSArDa	202888	<ls>BHĀG. P. 5,15,30</ls>	     30 not found
82462	maDumEreya	340902	<ls>BHĀG. P. 6,2,59</ls>     59 not found
84707	mahABogin	350359	<ls>BHĀG. P. 5,24,81</ls>    81 not found
206319	apravizwa	588168	<ls n="BHĀG. P.">7,49,4</ls> 49 not found
```

**Comment by Andhrabharati** [comment]
@funderburkjim,

Here are the corrections--
```
21824	ekapati	84708	<ls>BHĀG. P. 4,26,27</ls>	     4,20,27
49440	daSArDa	202888	<ls>BHĀG. P. 5,15,30</ls>	     3,15,30
82462	maDumEreya	340902	<ls>BHĀG. P. 6,2,59</ls>     6,1,59
84707	mahABogin	350359	<ls>BHĀG. P. 5,24,81</ls>    5,24,31
206319	apravizwa	588168	<ls n="BHĀG. P.">7,49,4</ls> 2,9,34 (other places: 7,12,15 & 10,3,14)
```

**Comment by funderburkjim** [comment]
Got it.  How do you find these?  Do you have a digitization of the Burnouf?

**Comment by Andhrabharati** [comment]
Not of Burnouf. but I have the full BhP text itself of another edition; as I had mentioned earlier, not much diffrences between various editions of BhP.

**Comment by Andhrabharati** [comment]
The last one above (apravizwa-- 7,49,4) appears not to be from BhP (as other places do not come anywhere closer)!

**Comment by Andhrabharati** [comment]
BTW, I have the full text of MBh.(not the **erratic** BORI ed. as Marcis & many others widely quote being available publicly!) and R. (not the Baroda ed.) [and quite many other works] as well

**Comment by funderburkjim** [comment]
A digitization of BhP might be used for a 'validation' of pwg references to BHĀG. P.

**Comment by funderburkjim** [comment]
## BHĀG. P. links standardized
Work in [issue109 directory](https://github.com/sanskrit-lexicon/PWK/tree/master/pwkissues/issue109).
We started with 879 standard links.  
* 983  bhagp_standard_2.txt
*  54  bhagp_nonstandard_2.txt.  of form 'BHĀG. P. ed. ***'

**Comment by gasyoun** [comment]
> I have the full text

@Andhrabharati when you say text you mean a cleaned OCR?

> erratic BORI ed. as Marcis & many others widely quote being available publicly!

Oh, yes, it has up to a 10% error rate, very high indeed. But none has any other.

**Comment by Andhrabharati** [comment]
> > I have the full text
> 
> @Andhrabharati when you say text you mean a cleaned OCR?

No, not the cleaned OCR; but scraped-off from the actual published work!!

> > erratic BORI ed. as Marcis & many others widely quote being available publicly!
> 
> Oh, yes, it has up to a 10% error rate, very high indeed. But none has any other.

"But none has any other."  do you mean no one has any other source? if so, **I do not belong to that category!!**

**Comment by gasyoun** [comment]
> many people in India do not consider it to be any where near authentic (as against people from the other parts of the world).

And thy are fully right. I agree with them. But as the Russian translation is based on it, I do want to have BORI as well. It's misserably digitised now and none of the XIX century big ones are as I'm aware of. 

If one looks at https://github.com/vishvAsa/mahAbhAratam/tree/6fed1a5f82b7932435aad3bb7afeb926a62adf7e

most of it I do not understand or it is of no use for Cologne or me personally:

goraxapura-pAThaH Hindi and Kannada anuvadas is not what I'm hunting for

kumbhakoNa-prastutiH has only 3 kandas

kumbhakonam is scraped from https://sanskritdocuments.org/mirrors/mahabharata/mahabharata-sarit.html?

madhva-tAtparya-nirNayaH no idea what it is

shAstri-vAvilla - non Bombay, non Calcutta?

sukthankarAdi-pAThaH - it is an alternative digital edition of BORI?
